src/Firebase/Database.purs

```diff
@@ -12,3 +13,9 @@ foreign import onValue
    . String
   -> (Json -> Eff (firebase :: FIREBASE | eff) Unit)
   -> Eff (firebase :: FIREBASE | eff) (Eff (firebase :: FIREBASE | eff) Unit)
+
+foreign import write
+  :: ∀ eff
+   . String
+  -> Json
+  -> Eff (firebase :: FIREBASE | eff) Unit
```

Comment:
You *might* need `err :: EXCEPTION` here, as Firebase will not accept some JSON values (such as objects with keys with `/` in them). But I'm not 100% sure.